website/blog/2018-01-10-1.10.0.md

```diff
@@ -401,4 +401,4 @@ We've fixed up the behavior of the CLI when it comes to ignored files. Previousl
 [#3641]: https://github.com/prettier/prettier/pull/3641
 [#3643]: https://github.com/prettier/prettier/pull/3643
 [#3649]: https://github.com/prettier/prettier/pull/3649
-[plugin api]: https://prettier.io/docs/en/plugins
```

Comment:
it works without `.html`

Reply:
Yeah, I've noticed that, but I think `.html` should be better as it's a static site(?).

Reply:
🤷‍♂️